crates/connectors/ndc-postgres/src/schema.rs

```diff
@@ -233,6 +235,14 @@ pub async fn get_schema(
         })
         .collect();
 
+    let generated_procedures: Vec<models::ProcedureInfo> =
+        query_engine_translation::translation::mutation::generate::generate(&metadata.tables)
+            .iter()
+            .map(|(name, mutation)| mutation_to_procedure(name, mutation))
+            .collect();
+
+    procedures.extend(generated_procedures);
```

Comment:
We should probably consider what to do on name clashes with user defined native query procedures.

Reply:
Currently, any user-defined thing overrides the generated one. This seems the right way round. We can re-address this if we choose to store the generated stuff in metadata, in which case the user might manually rename things and we should help them not make confusing mistakes.